<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>upload_file</class>
 <widget class="QDialog" name="upload_file">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pb_find">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_file">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="pgb_set">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_upload">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始上传</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_food">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>美食</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_music">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>音乐</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_cartoon">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>220</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>动漫</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_game">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_life">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>生活</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clear">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
